operator/cilium_node.go

```diff
@@ -57,6 +60,11 @@ func startSynchronizingCiliumNodes(nodeManager allocator.NodeEventHandler) {
 				if node := k8s.ObjToCiliumNode(obj); node != nil {
 					// node is deep copied before it is stored in pkg/aws/eni
 					nodeManager.Create(node)
+					if wgOperator != nil {
```

Comment:
Once https://github.com/cilium/cilium/pull/15295 is merged, we can convert this logic to be inside its own handler. It's OK to do this for now